### DISCUSSION TITLE
Property 'step' does not exist on type 'StepsRouterBindProps'.

### DISCUSSION THREAD

**Discussion by KavyaBaskar12** [primefaces/community#326] [PrimeVue] [answered]
Hi, 
I am trying to use the props parameter of the item slot which is of type StepsRouterBindProps.
The example in  PrimeVue (https://primevue.org/steps/confirmation) uses v-bind="props.step".
However, the interface StepsRouterBindProps does not contain the property step.

Any update coming soon?

**Comment by tugcekucukoglu** [comment, marked as answer]
Hi,

`item` does not have `step` option. Available options are here: https://primevue.org/steps/confirmation/#api.steps.slots

**Reply by KavyaBaskar12** [reply]
As I mentioned, its the _props_ that should have _step_ option, according to the example. But it does not.
